metals/src/main/scala/scala/meta/internal/metals/MetalsServerConfig.scala

```diff
@@ -110,9 +110,7 @@ object MetalsServerConfig {
         )
       case "vim-lsc" =>
         base.copy(
-          // window/logMessage output is always visible and non-invasive in vim-lsc
-          statusBar = StatusBarConfig.logMessage,
-          // Not strictly needed, but helpful while this integration matures.
+          statusBar = StatusBarConfig.showMessage,
```

Comment:
I was testing this today and realized that with `logMessage` things are correctly getting logged, but the user is getting zero feedback about what it happening. I think it's much more preferable to have this set to `showMessage`.

Reply:
I use this some more today, and I'm going to revert this. While it's hard without getting feedback, with this being set this way, the user receives way too much. .